pages/docs/2_modify/advanced/linking_local_version.md

```diff
@@ -58,3 +57,37 @@ By following these steps, you can effectively manage local changes to the base C
 <div class="note">
 Linking multiple different development versions simultaneously will not work, as running <code>$lerna-linker linkSource</code> will overwrite all previously made links.
 </div>
+
+## Yarn Workspaces
```

Comment:
Could you move this section above lerna-linker?
As this may be the better solution in general.

Reply:
Done, it is now above the lerna-linker section. I also adjusted some whitespaces in the lerna-linker secton, because the code blocks had strange indentations on the page as a result.